python/setup.py

```diff
@@ -208,12 +208,18 @@ def build_extension(self, ext: CMakeExtension) -> None:
     # See issue #222: https://github.com/SINTEF/dlite/issues/222
     # extras_require={"all": extra_requirements},
     packages=["dlite"],
+    scripts=[
+        #str(SOURCE_DIR / "bindings" / "python" / "scripts" / "dlite-validate"),
+    ],
```

Comment:
```suggestion
```
If it's not used, it shouldn't be included.

If what you wish to do is to have callable scripts upon installation, use the `console_scripts` key in the `entry points` dictionary ([setuptools documentation](https://setuptools.pypa.io/en/latest/userguide/entry_point.html)).

Reply:
It awaits PR #461 to be merged. When that is done, it should be uncommented.